apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -97,6 +114,79 @@ class ConnectionInstructions extends React.Component {
   }
 }
 
+class CPExpressInstallInstructions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // TODO: layout with images
+    // TODO: get screenshot for image copy
+    // TODO: reset button gif- put in our cdn?
+    // TODO: make i18n strings for all the strings
+    return (
+      <div id="CPExpressInstallInstructions">
+        <h2>Install the Firmata firmware onto the CP Express</h2>
+        <ol>
+          <li>
+            Set your Circuit Playground Express to Bootloader Mode
+            <div style={style.oneColumn}>
+              <ul>
+                <li>
+                  Press or double-press the RESET button in the center of the
+                  board.
+                </li>
+                <li>
+                  You've successfully entered bootloader mode when the LEDs turn
+                  green and your computer detects a new removable storage device
+                  called 'CPLAYBOOT'.
+                </li>
+                <li>
+                  If pressing RESET once doesn't work, try double-pressing it.
+                </li>
+                <li>
+                  If the color LEDs turn all red, check your USB cable, try
+                  another cable or another USB port.
+                </li>
+              </ul>
+              <img
+                src="https://cdn-learn.adafruit.com/assets/assets/000/051/201/original/makecode_uf2.gif?1519151701"
+                width={200}
+                alt={
+                  'GIF showing two ways to enter bootloader mode on the Circuit Playground Express by pressing the reset button once, then twice in a row.'
+                }
+              />
+            </div>
+          </li>
+          <li>
+            Copy over the Firmata firmware.
+            <ul>
+              <li>
+                <SafeMarkdown
+                  markdown={`Download the [Circuit Playground Express Firmata](${CIRCUIT_PLAYGROUND_EXPRESS_FIRMATA_URL})`}
+                />
+              </li>
+              <li>
+                Copy the '{CIRCUIT_PLAYGROUND_EXPRESS_FIRMATA_FILENAME}' file to
```

Comment:
Here are a couple screenshots for copying over .UF2 file- one for mac OS and one for windows.

mac:

![mac](https://github.com/code-dot-org/code-dot-org/assets/107423305/d3364911-7121-4161-ac05-0a7dd4aefc12)

windows:

<img width="1280" alt="windows" src="https://github.com/code-dot-org/code-dot-org/assets/107423305/40bead68-ec73-4d62-9e1f-ac86c962682b">

Reply:
TY! Note to self: look at history for where we were hosting those other images that we had previously shown on this page to find out where to upload these